stable/sonatype-nexus/templates/statefulset.yaml

```diff
@@ -146,17 +153,34 @@ spec:
           secret:
             secretName: {{ template "nexus.proxy-ks.name" . }}
         {{- end }}
+        {{- if not .Values.persistence.enabled }}
         - name: nexus-data
-          {{- if .Values.persistence.enabled }}
-          persistentVolumeClaim:
-            claimName: {{ .Values.persistence.existingClaim | default (printf "%s-%s" (include "nexus.fullname" .) "data") }}
-          {{- else }}
           emptyDir: {}
           {{- end }}
+        {{- if not .Values.nexusBackup.persistence.enabled }}
         - name: nexus-backup
-          {{- if and .Values.nexusBackup.enabled (.Values.nexusBackup.persistence.enabled) }}
-          persistentVolumeClaim:
-            claimName: {{ .Values.nexusBackup.persistence.existingClaim | default (printf "%s-%s" (include "nexus.fullname" .) "backup") }}
-          {{- else }}
           emptyDir: {}
           {{- end }}
+{{- if or .Values.persistence.enabled .Values.nexusBackup.persistence.enabled }}
+  volumeClaimTemplates:
```

Comment:
so by using `volumeClaimTemplates` we're loosing the ability to reuse an existing disk/claim

----

maybe the solution to your issue with AWS and multi-zone disks is to instead first create the disks/claims before applying the helm chart